sbt-typesafe-conductr-tester/project/plugins.sbt

```diff
@@ -0,0 +1,2 @@
+lazy val root = Project("plugins", file(".")).dependsOn(plugin)
+lazy val plugin = file("../").getCanonicalFile.toURI
```

Comment:
If you cd to an scripted test then target meta data will be created that can upset scripted. You should not do that.

> On 3 Apr 2015, at 11:42 pm, Konrad Malawski <notifications@github.com> wrote:
> 
> In sbt-typesafe-conductr-tester/project/plugins.sbt:
> 
> > @@ -0,0 +1,2 @@
> > +lazy val root = Project("plugins", file(".")).dependsOn(plugin)
> > +lazy val plugin = file("../").getCanonicalFile.toURI
> I'd usually just cd into one of src/sbt-test/... and test there, but no prob if you want the tester around.
> 
> —
> Reply to this email directly or view it on GitHub.
>